sql/telemetry/feature_usage/query.sql

```diff
@@ -0,0 +1,114 @@
+SELECT
+  events_t.client_id AS clients_id,
+  events_t.submission_date AS submission_date,
+  COALESCE(protection_report, 0) AS protection_report,
+  COALESCE(pinned_tab, 0) AS pinned_tab,
+  COALESCE(default_browser, 0) AS default_browser,
+  COALESCE(many_uri, 0) AS many_uri,
+  COALESCE(pip, 0) AS pip,
+  COALESCE(filled_password, 0) AS filled_password,
+  COALESCE(remember_password, 0) AS remember_password,
+  COALESCE(sync_configured, 0) AS sync_configured
+FROM
+  (
+    SELECT
+      client_id,
+      submission_date,
+      IF(event_method = 'show' AND event_object = 'protection_report', 1, 0) AS protection_report,
+      IF(event_category = 'pictureinpicture' AND event_method = 'create', 1, 0) AS pip,
+    FROM
+      `moz-fx-data-shared-prod.telemetry.events`
+    WHERE
+      submission_date = @submission_date
+      AND normalized_channel = "release"
```

Comment:
Are we sure we want to restrict this to just release rather than allowing all channels as we do for KPIs?

Reply:
I had been restricting my analysis to release, but this table may be useful for other applications, so I'll drop the restriction.